### PR TITLE
Updated Vagrantfile

### PR DESCRIPTION
**Pull request by djosborne**

### CODE REVIEW

File: vagrant-centos/units/marathon-lb.service [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+[Service]
+ExecStartPre=-/usr/bin/docker kill marathon-lb
+ExecStartPre=-/usr/bin/docker rm marathon-lb
+#EnvironmentFile=/etc/sysconfig/calico
+ExecStart=/usr/bin/docker run --name marathon-lb \
+	--privileged \
+	--net=host \
+	-e PORTS=9090 \
+	djosborne/marathon-lb sse --marathon http://172.24.197.101:8080 --group external
```
Comment (djosborne) [inline vagrant-centos/units/marathon-lb.service]: Need to make this a variable...

File: Vagrantfile [hunk 2]
```diff
@@ -1,145 +0,0 @@
-# -*- mode: ruby -*-
-# vi: set ft=ruby :
```
Comment (robbrockbank) [inline Vagrantfile]: I don't think you can delete this until the docs that reference this have been updated.

Or, update the referencing docs :-)

My take, given that we are temporarily reworking the docs that will eventually go into calico containers is to put a new-docs directory in this repo that will basically be copied to calico-containers/docs when we are done.

Comment (djosborne) [inline Vagrantfile]: Agreed that we should update the reference docs. but I just want to make sure we're on the same page - My take was that this repo is effectively now the `calico-containers/docs/mesos` directory. Therefore we should open a pr that is simply `mv calico-mesos-deployments/docs/* calico-mesos-deployments`. Right?

**Comment by djosborne** [comment]
Vagrant changes were moved to calico-containers' new mesos section: https://github.com/projectcalico/calico-containers/tree/master/docs/mesos

**Pull request closed without merging**